processing/src/main/java/org/apache/druid/query/Query.java

```diff
@@ -170,9 +169,4 @@ default String getSqlQueryId()
   {
     return withOverriddenContext(ImmutableMap.of(QueryContexts.LANE_KEY, lane));
   }
-
-  default VirtualColumns getVirtualColumns()
```

Comment:
I'm assuming this change has nothing to do with the bug fix correct?

Reply:
It was something failing inspections since all the implementations of Query implement that method, it's only needed for the old rewrite mode (so I added this back in)